autoconfigure/storage-applicationinsights/src/main/resources/zipkin-server-applicationinsights.yml

```diff
@@ -0,0 +1,7 @@
+# When enabled, this allows shorter env properties (ex -Dspring.profiles.active=applicationinsights)
+zipkin:
+  storage:
+    applicationinsights:
+      instrumentationKey: ${AI_INSTRUMENTATION_KEY:}
```

Comment:
It is readable here in config file.. but during actual configuration user jus sets AI_INSTRU... and it would be much clearer to have prefix.